drivers/adc/adc_ite_it8xxx2.c

```diff
@@ -91,36 +100,51 @@ static int adc_it8xxx2_channel_setup(const struct device *dev,
 		return -EINVAL;
 	}
 
-	if (channel_cfg->channel_id < CHIP_ADC_CH4) {
+	/* The channel is set to ADC alternate function */
+	pinmux_pin_set(porti, channel_cfg->channel_id, IT8XXX2_PINMUX_FUNC_1);
```

Comment:
Add a paragraph here. Set the corresponding channel to ADC alternate function in the channel setup.